keras_nlp/models/t5/t5_presets.py

```diff
@@ -0,0 +1,137 @@
+            "path": "t5",
+            "model_card": "https://github.com/google-research/text-to-text-transfer-transformer/blob/main/released_checkpoints.md",
+        },
+        "config": {
+            "use_gated_activation": False,
+            "activation": "relu",
+            "hidden_dim": 2048,
+            "key_value_dim": 64,
+            "output_dim": 512,
+            "dropout": 0.1,
+            "layer_norm_epsilon": 1e-06,
+            "num_heads": 8,
+            "num_blocks": 6,
+            "vocabulary_size": 32128,
+        },
+        "preprocessor_config": {},
+        "weights_url": "TODO",
+        "weights_hash": "TODO",
+        "vocabulary_url": "TODO",
+        "vocabulary_hash": "TODO",
+        "merges_url": "TODO",
+        "merges_hash": "TODO",
+    },
+    "t5_base": {
+        "metadata": {
+            "description": (
+                "'Base' configuration of T5 (223 million parameters)."
+            ),
+            "params": 222903552,
+            "official_name": "T5-Base",
+            "path": "t5",
+            "model_card": "https://github.com/google-research/text-to-text-transfer-transformer/blob/main/released_checkpoints.md",
+        },
+        "config": {
+            "use_gated_activation": False,
+            "activation": "relu",
+            "hidden_dim": 3072,
+            "key_value_dim": 64,
+            "output_dim": 768,
+            "dropout": 0.1,
+            "layer_norm_epsilon": 1e-06,
+            "num_heads": 12,
+            "num_blocks": 12,
+            "vocabulary_size": 32128,
+        },
+        "preprocessor_config": {},
+        "weights_url": "TODO",
+        "weights_hash": "TODO",
+        "vocabulary_url": "TODO",
+        "vocabulary_hash": "TODO",
+        "merges_url": "TODO",
+        "merges_hash": "TODO",
+    },
+    "t5_large": {
+        "metadata": {
+            "description": (
+                "'Large' configuration of T5 (738 million parameters)."
+            ),
+            "params": 737668096,
+            "official_name": "T5-Large",
+            "path": "t5",
+            "model_card": "https://github.com/google-research/text-to-text-transfer-transformer/blob/main/released_checkpoints.md",
+        },
+        "config": {
+            "use_gated_activation": False,
+            "activation": "relu",
+            "hidden_dim": 4096,
+            "key_value_dim": 64,
+            "output_dim": 1024,
+            "dropout": 0.1,
+            "layer_norm_epsilon": 1e-06,
+            "num_heads": 16,
+            "num_blocks": 24,
+            "vocabulary_size": 32128,
+        },
+        "preprocessor_config": {},
+        "weights_url": "TODO",
+        "weights_hash": "TODO",
+        "vocabulary_url": "TODO",
+        "vocabulary_hash": "TODO",
+        "merges_url": "TODO",
+        "merges_hash": "TODO",
+    },
+    "t5_3b": {
+        "metadata": {
+            "description": (
+                "'3B' configuration of T5 (2.85 billion parameters)."
```

Comment:
We will already show the parameter count in our table on keras.io, so might want to make the description more orthogonal. You can see our current descriptions here https://keras.io/api/keras_nlp/models/#backbone-presets

Reply:
There isn't that much useful info we can bring up in the one-line description -- I included the number of layers and a more user-readable parameter count, which is ultimately the defining characteristic of each checkpoint.